repo/gradle-settings-conventions/develocity/src/main/kotlin/develocity.settings.gradle.kts

```diff
@@ -11,21 +11,18 @@ if (buildProperties.buildScanServer != null) {
     plugins.apply("com.gradle.common-custom-user-data-gradle-plugin")
 }
 
-gradleEnterprise {
+develocity {
     buildScan {
         if (buildScanServer != null) {
             server = buildScanServer
-            publishAlways()
 
             capture {
-                isTaskInputFiles = true
-                isBuildLogging = true
-                isBuildLogging = true
-                isUploadInBackground = true
+                buildLogging = true
+                uploadInBackground = true
```

Comment:
this should be conditioned on being a local build. but this is not a regression.

Reply:
I leave it to the maintainers to decide whether to introduce this change here or not